src/validation_workflow/docker/validation_docker.py

```diff
@@ -61,7 +61,7 @@ def start_cluster(self) -> bool:
     def validation(self) -> bool:
         # STEP 2 . inspect image digest between opensearchproject(downloaded/local) and opensearchstaging(dockerHub)
         if not self.args.using_staging_artifact_only:
-            self.image_names_list = [self.args.OS_image, self.args.OSD_image]
+            self.image_names_list = ['opensearchproject/' + project for project in self.args.projects]
             self.image_names_list = [x for x in self.image_names_list if (os.path.basename(x) in self.args.projects)]
```

Comment:
what does these two lines of `self.image_name_list` do?

Reply:
Second this.

Reply:
I have unassigned the args self.args.OS_image, self.args.OSD_image in args.py  and declared a list which stores the images names based projects=["opensearch", "opensearchdashboards"]

Reply:
thanks, Divya, I just feel a bit redundant here becasue you constructed the image_names_list from args.projects on the first line of code, and then you do a filtering from the same args.projects on the next line of code. Could you give me an exmaple of expected output from the 2nd line of code assuming the image_names_list has ["opensearchproject/opensearch", "opensearchproject/opensearchdashboards"]?

Reply:
I get it now. Sorry missed this line while editing. It's of no use. Will remove it Thanks for pointing.